### PR TITLE
Two sections comparing Boost.Asio Timers with JS setTimeout

### PR DESCRIPTION
**Pull request by semmel**

### CODE REVIEW

**Comment by JohnMurray** [comment]
Hey, sorry for taking so long. I'll take a look at it this weekend.

File: versus/js/05-asynchronous-computation/nested-timeouts-description.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		creating it on the heap and manage access and lifetime by a C+ shared pointer created on the stack.
+		In order to prevent it's reference count drop to zero and get the timer object destroyed nevertheless
+		when the block scope exits, we have to pass a copy of it into the scheduled timeout handler.
+	</p>
+	<p>
+		This looks awkward because the timer object is not really used in it's completion handler. While JavaScript
+		manages the lifetime of it's reference objects liker timers itself, in C++ the reference counting is left to
+		the application.
+	</p>
+</p>
```
Comment (JohnMurray) [inline versus/js/05-asynchronous-computation/nested-timeouts-description.html]: No need to next `<p>` tags here. may create non-uniform spacing with other posts.

**Comment by JohnMurray** [comment]
Hey @semmel, Just curious if you've had time to review my comments yet.

**Comment by semmel** [comment]
Hey @JohnMurray,
I've addressed the missing yaml spaces and concretised what I meant with `Function.prototype.bind`. That part was really not well explained.
Considering `std::bind` in C++ however, I don't feel qualified to explain that better than any textbook in C++ does. And I would refer the reader to look it up him/herself.

Also I would like to keep the length of code and text both short and in balance. Kind of *cookbook* or *best-practise*-like.

**Pull request merged**